### PR TITLE
Fix/elapsed time

### LINKED ISSUE #90: [BUG] On the result screen, should be time taken to complete
**Describe the bug**
When the user ends a test in Exam Mode, the congratulation text on the end screen displays the remaining time instead of the time taken to complete the test.

**To Reproduce**
1. Go to 'Exam Mode'
2. Click on 'Begin'
3. Answer questions
4. See end screen

**Expected behavior**
On the end screen, the congratulatory text should include the time taken to complete test.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by RKOsvk**
Fixes #90 

- [x] I have read and agree to the [Code of Conduct](https://github.com/eduardconstantin/azure-fundamentals/blob/main/CODE_OF_CONDUCT.md), and have followed the [Contributing guidelines](https://github.com/eduardconstantin/azure-fundamentals/blob/main/CONTRIBUTING.md).

## Pull Request Type

- [x] Bug fix

### CODE REVIEW

File: pages/exam/index.tsx [hunk 1]
```diff
@@ -68,6 +68,12 @@ const Exam: NextPage = () => {
     }
   }, [checkPassed, remainingTime]);
 
+  const totalSeconds = 3 * 60;
```
Comment (eduardconstantin) [inline pages/exam/index.tsx:71]: why is this hardcoded?

Comment (RKOsvk) [reply pages/exam/index.tsx:71]: if you want i can change this. for something like this 
const totalTimeInSeconds = minutes * 60 + seconds;

Comment (eduardconstantin) [reply pages/exam/index.tsx]: the time of the exam may vary, can be 15 minutes or less, I put 3 minutes to test it

Comment (RKOsvk) [reply pages/exam/index.tsx]: i changed it and made another pull request it is not fixed anymore

Comment (eduardconstantin) [reply pages/exam/index.tsx]: why did you made another PR?

Comment (RKOsvk) [reply pages/exam/index.tsx]: you dont want it hardcoded so i changed it

Comment (eduardconstantin) [reply pages/exam/index.tsx]: oh, you made another commit, you can set the pr ready for review

Comment (RKOsvk) [reply pages/exam/index.tsx]: yeah sorry for confusion

**Review by eduardconstantin — APPROVED** [review]
ok, looks good! Thank you for fixing this bug!

**Pull request merged**